tools/physiofit/physiofit.xml

```diff
@@ -1,278 +1,98 @@
-
-                </when>
-            </conditional>
-
-            <conditional name='fm_select'>
-                <param name='fm' type='select' label='Select yes to add bounds on metabolite fluxes'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='flux_met_bounds' type='text' value='()' label='Bounds on metabolite fluxes. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <conditional name='cb_select'>
-                <param name='cb' type='select' label='Select yes to add bounds on biomass concentrations'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='conc_biom_bounds' type='text' value='()' label='Bounds on biomass concentrations. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <conditional name='fb_select'>
-                <param name='fb' type='select' label='Select yes to add bounds on biomass fluxes'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='flux_biom_bounds' type='text' value='()' label='Bounds on biomass fluxes. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <param name='verbose' type='boolean' argument='--verbose' truevalue='-v' falsevalue='' label='Flag to get debug information'/>
-
-        </section>
-
+        <param name='datafile' type='data' format='tabular, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+        <param name='configfile' type='data' format='yaml' multiple='true' label='Upload yaml configuration file containing run parameters'/>
     </inputs>
-
     <outputs>
-    <data name='plots' label='Plots' format='pdf'/>
-    <data name='fluxes' label='Flux_results' format='tabular'/>
-    <data name='stats' label='Stat_results' format='tabular'/>
-    <data name='config' label='Config_file' format='json'/>
-    <data name='log' label='Run_Log' format='txt'/>
+        <!-- <data name='config' label='Config_file' format='yaml'/> -->
+        <data name='recap' label='Summary file' format='csv'/>
+        <data name='zip' label='Results' format='zip'/>
+        <data name='log' label='Log' format='txt'/>
+        <!-- <collection name='results' type='list' label='Resulting files'>
+            <discover_datasets pattern='__name_and_ext__'/>
+        </collection> -->
```

Comment:
The collection seems a better solution than zip, why give up?

Reply:
I tried to but the discover_datasets is not able to detect the generated files!